src/Helper/Log/TelemetryEventPropertiesBuilder.php

```diff
@@ -86,38 +86,12 @@ public function build(TelemetryEventNameEnum $name): array {
    */
   protected function buildTravisCiJobProperties(): array {
```

Comment:
Lingering reference to Travis CI in this method name: `buildTravisCiJobProperties()`.  Why don't you do one more search around the codebase to make sure nothing else was missed. Use a different method this time, like PhpStorm's search, in case the original one missed anything (if the `grep` command was naive, for example).

Reply:
Only other travis ci references are present in Telemetry related code which will be removed in a separate PR